README.md

```
@@ -19,17 +19,17 @@ Please ensure these environment variables are set before running the script.
-yarn install
+pnpm install
 ```
 
-Then, to run the script using Yarn, use the `yarn generate` command.
+Then, to run the script using pnpm, use the `pnpm generate` command.
 
 ### As a dependency in another project
 
 Install the package as a dependency in your project:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)